burn-tch/src/ops/module.rs

```diff
@@ -163,6 +163,53 @@ impl<E: TchElement> ModuleOps<TchBackend<E>> for TchBackend<E> {
         TchTensor::new(tensor)
     }
 
+    fn max_pool1d(
+        x: TchTensor<E, 3>,
+        kernel_size: usize,
+        stride: usize,
+        padding: usize,
+    ) -> TchTensor<E, 3> {
+        let tensor = tch::Tensor::max_pool1d(
+            &x.tensor,
+            kernel_size as i64,
+            stride as i64,
+            padding as i64,
+            [1, 1],
+            false,
+        );
+
+        TchTensor::new(tensor)
+    }
+
+    fn max_pool1d_with_indices(
+        x: TchTensor<E, 3>,
+        kernel_size: usize,
+        stride: usize,
+        padding: usize,
+    ) -> MaxPool1dWithIndices<TchBackend<E>> {
+        let (tensor, indices) = tch::Tensor::max_pool1d_with_indices(
+            &x.tensor,
+            kernel_size as i64,
+            stride as i64,
+            padding as i64,
+            [1, 1],
+            false,
+        );
+
+        MaxPool1dWithIndices::new(TchTensor::new(tensor), TchTensor::new(indices))
+    }
+
+    fn max_pool1d_with_indices_backward(
+        _x: TchTensor<E, 3>,
+        _kernel_size: usize,
+        _stride: usize,
+        _padding: usize,
+        _output_grad: TchTensor<E, 3>,
+        _indices: TchTensor<i64, 3>,
+    ) -> MaxPool1dBackward<TchBackend<E>> {
+        todo!("Not implemented by tch-rs");
+    }
```

Comment:
You should not override it. A default using the backward pass of max pool 2d should exists.